CHANGELOG.rst

```diff
@@ -4,14 +4,14 @@ Changelog
 This document describes changes between each past release.
 
 
-2.9.0 (unreleased)
+2.9.0 (2015-10-27)
 ------------------
 
 **New features**
 
 - Added notifications (#488)
```

Comment:
I would say that this feature deserves a bit of details in the changelog! 

What about *Added Pyramid events, triggered when the content of a resource has changed* ?